index.html

```diff
@@ -438,6 +438,27 @@ <h2>Data Model</h2>
 <a>verification methods</a>.
       </p>
 
+      <p>
+All of the data model terms and types in this specification map to URLs. The
+vocabulary where these terms are defined is the [[?SECURITY-VOCABULARY]]. The
```

Comment:
```suggestion
vocabulary where these URLs are defined is the [[?SECURITY-VOCABULARY]]. The
```

I am just being cautious. The (abstract) ***term*** is _defined in this spec_, the vocabulary does not "define" anything else than the URLs and the RDF related ontology statements. I am picky, but want to avoid the discussions about the normative nature of the definitions...